src/tower.py

```diff
@@ -0,0 +1,21 @@
+"""
+tower class
+"""
+
+
+class Tower:
+    """
+    tower class
+    with all levels
+    """
+    def __init__(self):
+        self.levels = []
+
+    def print(self, window):
+        """
+        print all levels
+        :param window: window to print
+        :return:
+        """
+        for level in self.levels:
+            level.print(window)
```

Comment:
Werden hier nicht einfach alle Level übereinander geschrieben? Wann müssen wir denn mehr als ein Level auf einmal ausgeben?

Reply:
Ich wollte eigentlich im tower garkeine print haben. PyLint hat aber gemeckert weil es nicht genügen public methonden gab

Reply:
habe aber schon ne idee für einen Fix.

Reply:
Naja, wenn du keine Methoden für den Tower hast, dann muss es keine Klasse sein

Reply:
Ja das Stimmt natürlich. 
Das Spiel haben wir aber auch nicht voll durchgeplant da könnte sich auch noch Nutzen ergeben,, ich denke die Klasse ist schon in Ordnung. habe das jetzt auch umgebaut, man kann jetzt eine gewünschte Ebene über den Index printen.